src/main/java/appeng/core/AppEng.java

```diff
@@ -177,7 +177,7 @@ private void postInit( FMLPostInitializationEvent event )
 		AEConfig.instance.save();
 
 		NetworkRegistry.INSTANCE.registerGuiHandler( this, GuiBridge.GUI_Handler );
-		NetworkHandler.instance = new NetworkHandler( "AE2" );
+//		NetworkHandler.instance = new NetworkHandler( "AE2" );
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3ACommentedOutCodeLine)